Base/usr/share/man/man1/xargs.md

```diff
@@ -0,0 +1,33 @@
+## Name
+
+xargs - build and execute commandlines from input
+
+## Synopsis
+
+```**sh
+$ xargs [options...] command [initial-arguments...]
+```
+
+## Description
+
+`xargs` reads items from a stream, delimited by some blank character (`delimiter`), and executes the `command` as many times as there are items, with any processed `initial-arguments`, possibly followed by an item read from the input.
+Each argument in `initial-arguments` is processed by replacing any occurence of the `replacement string` with the input item, and treating the entire resulting value as _one_ argument.
+If no argument in `initial-arguments` contains the `replacement string`, an argument is added at the end of the list containing only the `replacement string`.
+`xargs` will read the items from stdin by default, and when data is read from stdin, the standard input of `command` is redirected to `/dev/null`.
+The standard input is left as-is if data is read from a file.
```

Comment:
These two lines sound like they should be in a separate paragraph, don't you think?

Reply:
15,16?
I wouldn't know, I've never written a manpage before